C++/Logger/conversion/Stringifiable.h

```diff
@@ -2,17 +2,20 @@
 #define _STRINGIFIABLE_H_
 
 #include "ConvertibleTo.h"
+#include "../metaprogramming/value_type_is.h"
 
 #include <string>
 
 namespace Toolbox::Log::Conversion
 {
 
-class Stringifiable : public ConvertibleTo<std::string>
+class Stringifiable
+  : public ConvertibleTo<std::string>
+  , value_type_is <std::string>
 {
 public:
   virtual const std::string toString (void) const = 0;
-  operator std::string (void) const override final;
```

Comment:
Putting const after override is invalid syntax.  'const' qualifier may not appear after the virtual specifier 'override'.  Override helps you to identify subclasses that rely on that method and override it.  That way if you change the higher level interface, the classes that override it will not compile until you update them.